src/bokeh/resources.py

```diff
@@ -214,7 +216,7 @@ def verify_sri_hashes() -> None:
         raise ValueError("verify_sri_hashes() can only be used with full releases")
 
     from glob import glob
-    paths = glob(join(bokehjsdir(), "js/bokeh*.js"))
+    paths = [ Path(p) for p in glob(settings.bokehjsdir() / "js" / "bokeh*.js") ]
```

Comment:
FYI `Path` has a `glob` method on it https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob so maybe
```suggestion
 paths = list(settings.bokehjsdir().glob("js/bokeh*.js"))
```